.eslintrc.js

```diff
@@ -0,0 +1,76 @@
+module.exports = {
+    extends: ['airbnb', 'prettier', 'prettier/flowtype', 'prettier/react'],
+    env: {
+        browser: true,
+    },
+    plugins: ['prettier', 'flowtype'],
+    rules: {
+        'react/jsx-filename-extension': ['warn', { extensions: ['.js'] }],
+        'react/sort-comp': [
+            'warn',
+            {
+                order: [
+                    'static-methods',
+                    'lifecycle',
+                    'render',
+                    'everything-else',
+                ],
+            },
+        ],
+        'flowtype/define-flow-type': 'warn',
+        'flowtype/require-parameter-type': 'warn',
+        'flowtype/use-flow-type': 'warn',
+
+        // eslint-config-prettier rules that might cause problems
+        // see: https://github.com/prettier/eslint-config-prettier#special-rules
+        curly: ['error', 'multi-line'],
+        'max-len': [
+            'error',
+            80,
+            4,
+            {
+                ignoreUrls: true,
+                ignoreComments: false,
+                ignoreRegExpLiterals: true,
+                ignoreStrings: true,
+                ignoreTemplateLiterals: true,
+            },
+        ],
+        'no-confusing-arrow': [
+            'error',
+            {
+                allowParens: true,
+            },
+        ],
+        'no-mixed-operators': [
+            'off',
+            {
+                allowSamePrecedence: true,
```

Comment:
As soon as there are no objections to removing these we can. For the sake of the pull request it seemed easier